From 92ca154ce7d988a4087b276a748394ab79b98c13 Mon Sep 17 00:00:00 2001
Message-ID: <92ca154ce7d988a4087b276a748394ab79b98c13.1699403993.git.sharlatanus@gmail.com>
In-Reply-To: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
References: <0117df90b1ef43a67d4fa86e5b58e60f7c0a52e5.1699403992.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 29 Oct 2023 22:25:00 +0000
Subject: [PATCH 51/67] gnu: go-github-com-aws-aws-sdk-go-v2-service-sts: Move
 to (gnu packages golang-web).

* gnu/packages/golang.scm (go-github-com-aws-aws-sdk-go-v2-service-sts): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: I3503dbde93a46ff47dcc035243a7f320b7090794
---
 gnu/packages/golang-web.scm | 11 +++++++++++
 gnu/packages/golang.scm     | 10 ----------
 2 files changed, 11 insertions(+), 10 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 4bc9c944cb..aadf8ac73e 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -203,6 +203,17 @@ (define-public go-github-com-aws-aws-sdk-go-v2-service-ssooidc
     (propagated-inputs
      (list go-github-com-aws-smithy-go))))
 
+(define-public go-github-com-aws-aws-sdk-go-v2-service-sts
+  (package
+    (inherit go-github-com-aws-aws-sdk-go-v2)
+    (name "go-github-com-aws-aws-sdk-go-v2-service-sts")
+    (version "1.17.7")
+    (arguments
+     '(#:import-path "github.com/aws/aws-sdk-go-v2/service/sts"
+       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
+    (propagated-inputs
+     (list go-github-com-aws-smithy-go))))
+
 (define-public go-github-com-aws-smithy-go
   (package
     (name "go-github-com-aws-smithy-go")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 9c102ad7f0..7b254db540 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -11173,16 +11173,6 @@ (define-public go-github-com-dvsekhvalnov-jose2go
     (home-page "https://github.com/dvsekhvalnov/jose2go")
     (license license:expat)))
 
-(define-public go-github-com-aws-aws-sdk-go-v2-service-sts
-  (package
-    (inherit go-github-com-aws-aws-sdk-go-v2)
-    (name "go-github-com-aws-aws-sdk-go-v2-service-sts")
-    (version "1.17.7")
-    (arguments
-     '(#:import-path "github.com/aws/aws-sdk-go-v2/service/sts"
-       #:unpack-path "github.com/aws/aws-sdk-go-v2"))
-    (propagated-inputs (list go-github-com-aws-smithy-go))))
-
 (define-public go-github-com-aws-aws-sdk-go-v2-service-s3
   (package
     (inherit go-github-com-aws-aws-sdk-go-v2)
-- 
2.41.0

